src/App.css

```diff
@@ -145,6 +145,34 @@ header p {
   color: #2d3748;
```

Comment:
⚖️ **Midlevel Developer**

In the CSS changes in `src/App.css` around line 145, we're introducing a new highlighted card style for total monthly income. Just a nitpick, but should the color palette match the primary theme color used elsewhere for better consistency? Also, maybe extract repeated styles into a shared class for maintainability.